server/common/admin-rest-client/src/main/java/org/kaaproject/kaa/server/common/admin/AdminClient.java

```diff
@@ -305,6 +305,14 @@ public TopicDto getTopic(String topicId) throws Exception {
         return entity.getBody();
     }
 
+    public List<TopicDto> getTopicsByApplicationToken(String applicationToken) throws Exception {
```

Comment:
All corresponding methods that use applicationId should be marked as Deprecated and contain reference to corresponding new method